src/System.Linq.Expressions/src/Resources/Strings.resx

```diff
@@ -337,7 +337,7 @@
     <value>Incorrect number of arguments for the given members </value>
   </data>
   <data name="LambdaTypeMustBeDerivedFromSystemDelegate" xml:space="preserve">
-    <value>Lambda type parameter must be derived from System.Delegate</value>
+    <value>Lambda type parameter must be derived from System.MulticastDelegate (and not System.MulticastDelegate itself)</value>
```

Comment:
The *be* got lost in my dithering between the the shorter and the longer.

I'll go with the shorter. The main thing is that it's correct, and if someone misinterprets it as meaning MulticastDelegate is allowed, it'll become clear when the exception is raised, whereas the current message would be at odds with it not working whatever way one reads it.